### DISCUSSION TITLE
API / function design of insight - version 1.0.0?

### DISCUSSION THREAD

**Discussion by strengejacke** [easystats/insight#178] [what's your opinion]
I wondered if we're happy with the function design in _insight_, so it will likely not change substantially in the (near) future... If so, we might think of getting our first package towards version 1.0.0 in this year? This will also show other package developers that they can rely on _insight_ in case they want to use it in their own packages...

**Comment by pdwaggoner** [comment]
Thanks for bringing this up @strengejacke . Was there something specific you were unhappy with re: insight's design?

**Comment by strengejacke** [comment]
no, there's no particular dissatisfaction, just thought that we might show outside people that our packages are getting "stable" :-)

**Comment by DominiqueMakowski** [comment]
I agree that insight has become pretty stable, and that we should aim for a 1.0 release this year. Maybe it'd make sense to release 1.0 after a first version of the easystats ecosystem is on CRAN (cf. modelbased and correlation) [I will be submitting modelbased as soon as CRAN reopens]
